.github/workflows/ci.yml

```diff
@@ -162,18 +162,19 @@ jobs:
       PYTEST_ADDOPTS: --color=yes
       GAMMAPY_DATA: /home/runner/work/gammapy/gammapy/gammapy-datasets
     steps:
-      - name: set up python3.7
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.7
       - name: checkout repo
         uses: actions/checkout@v2
+      - name: install mamba
+        uses: mamba-org/provision-with-micromamba@main
+        with:
+          extra-specs: |
+            python=3.7
       - name: install dependencies
         # oldest supported versions according to setup.cfg
         # TODO set astropy and numpy versions
         run: |
-          pip install astropy click==7.0 numpy pydantic==1.0 pyyaml==5.1 regions==0.5 scipy==1.2
-          pip install pytest pytest-astropy requests tqdm
+          mamba install click==7.0 pydantic==1.0 pyyaml==5.1 regions==0.5 scipy==1.2 astropy==4.3 numpy==1.18.1
+          mamba install requests tqdm
```

Comment:
why two commands here?

Reply:
fixed with micromaba actions, though still in draft stage..